apps/consent-das/src/routers/consentQuestions.ts

```diff
@@ -159,7 +159,7 @@ router.post('/', async (req, res) => {
 	const { id, isActive, category } = req.body;
```

Comment:
Can we also update the routers to expect the `id` as `entityId`? For example, with this route:

```suggestion
	const { consentQuestionId, isActive, category } = req.body;
```

Reply:
do you want the request bodies updated too?

Reply:
oh I'm blind haha never mind